osm_fieldwork/OdkCentral.py

```diff
@@ -806,6 +806,33 @@ def getSubmissionMedia(
         result = self.session.get(url, verify=self.verify)
         return result
 
+    def getSubmissionPhoto(
+        self,
+        projectId: int,
+        instanceID: str,
+        xform: str,
+        filename: str,
+    ):
+        """Fetch a specific attachment by filename from a submission to a form.
+
+        Args:
+            projectId (int): The ID of the project on ODK Central
+            instanceID(str): The ID of the submission on ODK Central
+            xform (str): The XForm to get the details of from ODK Central
+            filename (str): The name of the attachment for the XForm on ODK Central
+
+        Returns:
+            (bytes): The media data
+        """
+        url = f"{self.base}projects/{projectId}/forms/{xform}/submissions/{instanceID}/attachments/{filename}"
+        result = self.session.get(url, verify=self.verify)
+        if result.status_code == 200:
+            log.debug(f"fetched {filename} from Central")
+        else:
+            status = eval(result._content)
```

Comment:
It's generally a bad idea to use `eval` in Python - there is almost always a better way!

Also accessing a private method is generally bad too.

Can this be achieved using .json() or another method?

Reply:
In `getMedia` func too eval was used, so I thought to use the same, but you are right, `eval` is not always a safe option. will refactor for both functions.